app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,59 @@
+module Admin
+  class UsersController < ApplicationController
+
+    def index
+      @users = User.all
+      authorize @users
```

Comment:
This is not the right way to do this. Did you check the bundit documentation?

Reply:
Yes ma that's what the documentation says